src/pages/home/index.js

```diff
@@ -1,13 +1,26 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import '../../App.css'
+import './home.css'
+import '../../components/item-template/item-template.css'
+import ItemTemplate from '../../components/item-template/item-template'
+import ItemList from './items-list.js'
+import {ItemContext} from '../../context/items'
 
 
-function App() {
+
+function Home(props) {
+
+  const item = useContext(ItemContext)
+  
+
   return (
-    <div style={{ textAlign: 'center', display: 'flex', justifyContent: 'center', height: '100vh', flexDirection: 'column' }}>
-       Home page
+    <div className='Background'>
+      <h1 className='Title'>Welcome to the Store!</h1>
+      <div className='Item-container'>
+        {ItemList.map((items,) => <ItemTemplate image={items.image} title={items.title} price={items.price} onClick={() =>(props.history.push(/item/${items.id}), item.updateItem(items.id))}/>)}
```

Comment:
onClick={() => {
  props.history.push()
  item.updateItem()
}}
then just pass the arguments 
with the argument for props.history.push you'll want it to be a template string